app/scripts/templates/settings/recovery_codes.mustache

```diff
@@ -33,20 +33,32 @@
             <div class="link-options">
                 <a class="replace-codes-link">{{#t}}Generate new recovery codes{{/t}}</a>
             </div>
-            <div class="button-row recovery-code-options">
-                <a class="recovery-code-option download-codes">
-                    <div class="graphic graphic-recovery-codes-download"></div>
-                    <p class="name">{{#t}}Download{{/t}}</p>
-                </a>
-                <a class="recovery-code-option copy-codes">
-                    <div class="graphic graphic-recovery-codes-copy"></div>
-                    <p class="name">{{#t}}Copy{{/t}}</p>
-                </a>
-                <a class="recovery-code-option print-codes">
-                    <div class="graphic graphic-recovery-codes-print"></div>
-                    <p class="name">{{#t}}Print{{/t}}</p>
-                </a>
-            </div>
+
+            {{^isIos}}
+                <div class="button-row recovery-code-options">
+                    <a class="recovery-code-option download-codes">
```

Comment:
is there a simpler way we can just hide the unusable on iOS buttons selectively with a `{{^isIos}}` or too much work?

Reply:
Hrm this one is a bit more tricky because I wanted the button to be centered. I think it should be alright for now.

Reply:
ah cool 👍